### PR TITLE
Added build recipe for a5-miseq.

### PR DESCRIPTION
**Pull request by ysu20**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/a5-miseq/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+{% set version = "20160825" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name|lower }}-{{ version }}.tar.gz
+  url: https://sourceforge.net/projects/ngopt/files/a5_miseq_linux_20160825.tar.gz/download
```
Comment (bgruening) [inline recipes/a5-miseq/meta.yaml]: Can you remove the /download from the URL

Comment (ysu20) [reply recipes/a5-miseq/meta.yaml]: Done.

**Comment by ysu20** [comment]
@bgruening  would you mind merging this PR if you are happy with my change?

**Comment by bgruening** [comment]
@ysu20 thanks a lot!

**Pull request merged**